blog/template.hbs

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+      <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>{{{title}}}</title>
+
+<style>
+  :root {
+    --primary-blue: #0083c9;
+    --primary-white: #fff;
+    --primary-red: #df5738;
+    --dark-blue: #1065a3;
+    --light-red: #e85f40;
+  }
+
+  .light-red {
+    color: var(--light-red);
+  }
+  body {
+    box-sizing: content-box;
+    max-width: 50%;
+    width: 100%;
+    height: 100%;
+    margin-left: 25%;
+    margin-right: 25%;
+    min-width: 20px;
+    padding: 0;
+    font-family: "Helvetica Neue", sans-serif;
+    font-weight: bold;
+  }
+
+  body nav{
+    position: sticky; 
+    top: 0;
+    background: #fff;
+  }
+
+  body progress{
+      /* Positioning */
+  position: fixed;
+  left: 0;
+  top: 0;
+
+  /* Dimensions */
+  width: 100%;
+  height: 5px;
+
+  /* Reset the appearance */
+  -webkit-appearance: none;
+     -moz-appearance: none;
+          appearance: none;
+
+  /* Get rid of the default border in Firefox/Opera. */
+  border: none;
+
+  /* Progress bar container for Firefox/IE10+ */
+  background-color: transparent;
+
+  /* Progress bar value for IE10+ */
+  color: red;
+  }
+</style>
+
+
+<script
+  async
+  src="https://www.googletagmanager.com/gtag/js?id=G-BGXW90V3SE"
+></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+  gtag("js", new Date());
+  gtag("config", "G-BGXW90V3SE");
+</script>
+
+<body>
+  <script>
+  $(document).on('ready',function{
```

Comment:
this is jquery notation you want something like
```
document.addEventListener("DOMContentLoaded", function(event) {
    // Your code to run since DOM is loaded and ready
});
```
However I would recommend loading this script at the end, then all the elements are ready and you dont need this